ui/components/app/snaps/snap-ui-renderer/components/section.ts

```diff
@@ -0,0 +1,54 @@
+import { SectionElement, JSXElement, SectionProps } from '@metamask/snaps-sdk/jsx';
+import { getJsxChildren } from '@metamask/snaps-utils';
+import { NonEmptyArray } from '@metamask/utils';
+import {
+  Display,
+  FlexDirection,
+  BackgroundColor,
+  BorderRadius,
+  JustifyContent,
+} from '../../../../../helpers/constants/design-system';
+import { mapToTemplate } from '../utils';
+import { UIComponent, UIComponentFactory } from './types';
+
+function generateJustifyContent(alignment?: SectionProps['alignment']) {
+  switch (alignment) {
+    default:
+    case 'start':
+      return JustifyContent.flexStart;
+
+    case 'center':
+      return JustifyContent.center;
+
+    case 'end':
+      return JustifyContent.flexEnd;
+
+    case 'space-between':
+      return JustifyContent.spaceBetween;
+
+    case 'space-around':
+      return JustifyContent.spaceAround;
+  }
+}
+
+export const section: UIComponentFactory<SectionElement> = ({
```

Comment:
Wondering if this should simply use `box()` with a few alterations to add the border radius and padding? Just to not duplicate the logic between `section` and `box`.

We use a similar pattern here: https://github.com/MetaMask/metamask-extension/blob/8c38109505d3a875df97a8f2304ec6c5dbfb9aca/ui/components/app/snaps/snap-ui-renderer/components/field.ts#L73-L75

WDYT?

Reply:
Yeah, makes sense. Updated here: [d18771c](https://github.com/MetaMask/metamask-extension/pull/26718/commits/d18771c1c74865bf95916d9dd6cc8c192265b0ac)